tasks/task05/task05.py

```diff
@@ -0,0 +1,12 @@
+def create_user(name:str, user_name:str,  age:int = 42, **extra):
```

Comment:
`create_user(first_name: str, last_name: str, age:int = 42, **kwargs):`
This is much better to understand and `**kwargs` is better not to rename.

Reply:
ok